docs/contrib/06_new_project.md

```diff
@@ -0,0 +1,60 @@
+# Adding a project to pySDC (and automatic testing)
+
+When you do a project with pySDC we appreciate it if you merge it back to the main repository.
+We are committed to keeping your work reproducible and prevent it from fading into oblivion.
+To that end, please write extensive tests for your code and add them to the project.
+See the [contribution guide](./../../CONTRIBUTING.md) for general advice on testing etc.
+This guide will detail only how to add a project to pySDC.
+
+When adding a new project to pySDC, it needs to be tested.
+In order to run those tests as fast as possible, a few steps are needed, that are given below.
```

Comment:
I think we don't need the above two lines anymore. My goal was to elaborate on these points with the above text.

Reply:
Fine. Will be removed in next commit